libhb/qsv_common.c

```diff
@@ -2053,6 +2054,10 @@ int hb_qsv_param_default(hb_qsv_param_t *param, mfxVideoParam *videoParam,
         {
             param->videoParam->ExtParam[param->videoParam->NumExtParam++] = (mfxExtBuffer*)&param->codingOption2;
         }
+        if ((info->capabilities & HB_QSV_CAP_LOWPOWER_ENCODE) && (info->codec_id == MFX_CODEC_HEVC))
+        {
+            param->videoParam->mfx.LowPower = MFX_CODINGOPTION_ON;
```

Comment:
Use "on" by default for newer CPUs

Reply:
we already check codec_id for capabilities at
https://github.com/HandBrake/HandBrake/blob/c121bf7c797218d4ba90b23c2c31d0eea0c64692/libhb/qsv_common.c#L389

Reply:
Capabilities is ability to make encoding by specific HW.
This is runtime check when start encoding for particular selected codec info->codec_id. In case of AVC codec MFX_CODINGOPTION_ON will not be selected.